src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -1046,32 +1046,6 @@ def test_find_subscriptions_thru_username_non_password(self, mock_auth_context):
         # assert
         self.assertEqual([], subs)
 
-    @mock.patch('adal.AuthenticationContext', autospec=True)
-    @mock.patch('azure.cli.core._profile._get_authorization_code', autospec=True)
-    def test_find_subscriptions_with_invalid_authority_url(self, _get_authorization_code_mock, mock_auth_context):
```

Comment:
`adal.AuthenticationContext` will not be called, since `SubscriptionFinder` is using `lambda _, _1, _2: mock_auth_context`.

`mock_auth_context` can simply be created as a `MagicMock()`.